packages/testing/config/jest/api/jest.setup.js

```diff
@@ -10,6 +11,31 @@ const { defineScenario } = require('@redwoodjs/testing/dist/api')
 const { db } = require(path.join(getPaths().api.src, 'lib', 'db'))
 
 const DEFAULT_SCENARIO = 'standard'
+const TEARDOWN_CACHE_PATH = path.join(
+  getPaths().generated.base,
+  'scenarioTeardown.json'
+)
+let teardownOrder = []
+let originalTeardownOrder = []
+
+const configureTeardown = async () => {
+  const schemaModels = (await getSchemaDefinitions()).datamodel.models.map(
+    (m) => m.dbName || m.name
+  )
+
+  // check if pre-defined delete order already exists and if so, use it to start
+  if (fs.existsSync(TEARDOWN_CACHE_PATH)) {
+    teardownOrder = JSON.parse(fs.readFileSync(TEARDOWN_CACHE_PATH).toString())
```

Comment:
Maybe use require at the top of this file instead of JSON.stringifying every time? What do you think?